src/ralph/util/management/commands/make_demo_data.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pkgutil
+import textwrap
+from optparse import make_option
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from ralph.util import demo as demo_package
+from ralph.util.demo import registry, DemoRunner
+
+
+def get_list(option, opt_str, value, parser):
+    setattr(parser.values, option.dest, value.split(','))
+
+
+class Command(BaseCommand):
+    """Create a example data."""
+
+    help = textwrap.dedent(__doc__).strip()
+    # TODO: add database option
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--flush',
+            action='store_true',
+            dest='flush',
+            default=False,
+            help='Flush database before generate.'
+        ),
+        make_option(
+            '--fixtures',
+            type='string',
+            action='callback',
```

Comment:
there is no need to parse this list by hand (and use setattr :hear_no_evil:). Use sth like this:
```
make_option(
    '--fixtures',
    dest='fixtures',
    action='append',
    type='str',
    default=[],
    help=_('Fixtures to apply'),
)
```
Then you can call `ralph make_demo_data --fixtures fix1 fix2 fix3`